framework/Optimizers/Optimizer.py

```diff
@@ -185,7 +187,14 @@ def _readMoreXMLbase(self,xmlNode):
               except ValueError: self.raiseAnError(ValueError, "Unable to convert to float the intial value for variable "+varname+ " in trajectory "+str(trajInd))
             if self.optTraj == None:
               self.optTraj = range(len(self.optVarsInit['initial'][varname].keys()))
-
+      elif child.tag == "constant":
+        try:
```

Comment:
Would we be better off shrinking this "try" to only act on the part of the code that we want to catch?